cmd/sql/flow.go

```diff
@@ -317,6 +350,57 @@ func executeCmd(cmd *cobra.Command, args []string) error {
 	return nil
 }
 
+func executeDeployCmd(cmd *cobra.Command, args []string) error {
+	astroRuntimeVersion, err := getAstroDockerfileRuntimeVersion()
+	if err != nil {
+		return err
+	}
+	SQLCLIVersion, err := sql.GetPypiVersion(sql.AstroSQLCLIProjectURL)
+	if err != nil {
+		return err
+	}
+	requiredRuntimeVersion, requiredPythonSDKVersion, err := sql.GetPythonSDKComptability(sql.AstroSQLCLIConfigURL, SQLCLIVersion)
```

Comment:
To be future-proof, we can have a lower and upper threshold of supported versions, as mentioned in the comment
https://github.com/astronomer/astro-cli/pull/1007/files#r1063343197